### PR TITLE
Signup validations

### PR DESCRIPTION
**Pull request by YesicaNavaB**
## Summary

Signup validations

## Trello Card

https://trello.com/c/lwFcrF3X

### CODE REVIEW

File: app/services/usersDataBase.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const { user: User } = require('../models');
+
+exports.userAlreadyExists = email =>
+  User.findAndCountAll({
```
Comment (gonziesc) [inline app/services/usersDataBase.js]: you dont need and find and count all, just a find would be fine!

File: app/graphql/users/middlewares.js [hunk 2]
```diff
@@ -1,13 +1,37 @@
...
+const logger = require('../../logger'),
+  yup = require('yup'),
+  errors = require('../../errors'),
+  { regexEmail, regexAlphanumeric } = require('../../helpers/regex'),
+  { userAlreadyExists } = require('../../services/usersDataBase');
 
-const createUser = (resolve, root, args) => {
-  logger.info("Middleware for 'createUser' mutation");
-  // Add different actions that you want to be executed before your resolver, i.e: input validation or caching
+const schema = yup.object().shape({
```
Comment (gonziesc) [inline app/graphql/users/middlewares.js]: i'll move this logic to another place

File: app/services/usersDataBase.js (hunk 1 above)
Comment (YesicaNavaB) [reply app/services/usersDataBase.js]: ok!

File: app/graphql/users/middlewares.js (hunk 2 above)
Comment (YesicaNavaB) [reply app/graphql/users/middlewares.js]: ok!

**Review by gonziesc — APPROVED** [review]

**Pull request merged**